src/dataman/RowReader.h

```diff
@@ -89,6 +89,32 @@ class RowReader {
         folly::StringPiece row,
         std::shared_ptr<const meta::SchemaProviderIf> schema);
 
+    static VariantType getDefaultProp(const meta::SchemaProviderIf* schema,
+                                      const std::string& prop) {
+        auto& vType = schema->getFieldType(prop);
+        switch (vType.type) {
+            case nebula::cpp2::SupportedType::BOOL: {
+                return false;
+            }
+            case nebula::cpp2::SupportedType::INT: {
+                return static_cast<int64_t>(0);
+            }
+            case nebula::cpp2::SupportedType::VID: {
+                return static_cast<VertexID>(0);
+            }
+            case nebula::cpp2::SupportedType::FLOAT:
+            case nebula::cpp2::SupportedType::DOUBLE: {
+                return static_cast<double>(0.0);
+            }
+            case nebula::cpp2::SupportedType::STRING: {
+                return static_cast<std::string>("");
+            }
+            default:
+                LOG(FATAL) << "Unknown type: " << static_cast<int32_t>(vType.type);
```

Comment:
FATAL will crash, maybe other level is better.

Reply:
Ok.